dbt/adapters/dremio/credentials.py

```diff
@@ -83,26 +92,31 @@ def __pre_deserialize__(cls, data):
             data["cloud_host"] = None
         if "software_host" not in data:
             data["software_host"] = None
+
         if "database" not in data:
             data["database"] = None
         if "schema" not in data:
             data["schema"] = None
+
         if "datalake" not in data:
             data["datalake"] = None
         if "root_path" not in data:
             data["root_path"] = None
-        if "environment" not in data:
-            data["environment"] = None
+
         if "pat" not in data:
             data["pat"] = None
+
+        if "environment" not in data:
+            data["environment"]
```

Comment:
Seems like we're missing a "= None" here?

Reply:
Oh that's odd. Will do.